### ISSUE TITLE
Compile Story Points And Priority Binary Tree

### ISSUE THREAD

**Issue by ez4bk** [ez4bk/smarter-university-system-17#28]
Activity of Epic #45 
Calculate story points and generate a group-level priority binary tree.

**Comment by ez4bk** [comment]
@varungole While we allow multiple user stories with the same story points, how should we construct our binary tree out of it? Should we also make some prioritization for those with the same points? If so, I think it will be a trouble compiling the final tree. 
From my perspective, we can prioritize those with the same points in our own tree. And when we compile the final tree, we ignore those individual prioritization stuff and follow strictly the total story points we have. Cause I realized that it's kind of hard for the some total story points to be the same number. Even if we do end up with a few same total points, we can have a bit of discussion on them, which won't be too many.
What do you think?

**Comment by ez4bk** [comment]
@varungole I see. Same weight simply goes left in your binary tree. Very true, will notify other members. But did you also make your tree balanced? I couldn't see any dummy nodes in your tree, and I was wondering how did you pick R4 as root node.

**Comment by varungole** [comment]
Hi, sorry for a late response. Yes, In my tree, same weights simply goes to the left. I implemented this logic to handle requirements having the same story points. Yes dummy nodes aren't there in my tree and I am going to ask the professor regarding this. So, I chose any node with 4 story points to be the root node, which in this case was R4. I wonder if it matters that the tree is balanced or not as the functionality of the tree is to only sort the requirements based on priority.
